packages/cli/generators/discover/index.js

```diff
@@ -0,0 +1,272 @@
+path = require('path');
+const fs = require('fs');
+const ArtifactGenerator = require('../../lib/artifact-generator');
+const modelMaker = require('../../lib/model-discoverer');
+const debug = require('../../lib/debug')('discover-generator');
+const utils = require('../../lib/utils');
+const modelDiscoverer = require('../../lib/model-discoverer');
+
+module.exports = class DiscoveryGenerator extends ArtifactGenerator {
+  constructor(args, opts) {
+    super(args, opts);
+
+    this.option('dataSource', {
+      type: String,
+      alias: 'ds',
+      description: 'The name of the datasource to discover',
+    });
+
+    this.option('views', {
+      type: Boolean,
+      description: 'Boolean to discover views',
+      default: true,
+    });
+
+    this.option('schema', {
+      type: String,
+      description: 'Schema to discover, required for postgresql',
```

Comment:
I'm not sure if there's a need for "required for postgresql".  Thoughts?

Reply:
You're right, removed.  

I had it in there because the postgres connector will discover `pg_` tables (along with `public` tables) without a specified schema, on reconsideration, it's not worth warning about.